dev/io.openliberty.microprofile.metrics.internal.5.x.monitor_fat/fat/src/io/openliberty/microprofile/metrics/internal/monitor_fat/ComputedMetricsTest.java

```diff
@@ -0,0 +1,637 @@
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.security.SecureRandom;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Base64;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSession;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class ComputedMetricsTest {
+
+    private static Class<?> c = ComputedMetricsTest.class;
+
+    private static final String[] ALWAYS_EXPECTED_COMPUTED_METRICS = new String[]{
+            "cpu_processCpuUtilization_percent{mp_scope=\"vendor\",}",
+            "memory_heapUtilization_percent{mp_scope=\"vendor\",}",
+            // "gc_time_per_cycle_seconds{mp_scope=\"vendor\",name=\"global\",}",
+            // "gc_time_per_cycle_seconds{mp_scope=\"vendor\",name=\"scavenge\",}"
+    };
+
+    @Server("ComputedMetricsServer")
+    public static LibertyServer computedMetricsServer;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        trustAll();
+        addRequiredApplicationsToServer();
+        computedMetricsServer.saveServerConfiguration();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        if (computedMetricsServer != null && computedMetricsServer.isStarted()) {
+            computedMetricsServer.stopServer("");
+        }
+        computedMetricsServer.restoreServerConfiguration();
+    }
+
+    private static void addRequiredApplicationsToServer() throws Exception {
+        String testName = "addRequiredApplicationsToServer";
+        Log.info(c, testName, "------- Add JDBC/Connection Pool application ------");
+        ShrinkHelper.defaultDropinApp(computedMetricsServer, "testJDBCApp",
+                "io.openliberty.microprofile.metrics.internal.monitor_fat.jdbc.servlet");
+        Log.info(c, testName, "------- Added testJDBCApp to dropins -----");
+        Log.info(c, testName, "------- Add RESTful application ------");
+        ShrinkHelper.defaultDropinApp(computedMetricsServer, "testRESTApp",
+                "io.openliberty.microprofile.metrics.internal.monitor_fat.rest");
+        Log.info(c, testName, "------- Added testRESTapp to dropins -----");
+    }
+
+    /*
+     * Tests if the computed metrics using the base metrics are present in
+     * /metrics?scope=vendor endpoint output, as soon as, the server starts and is a
```

Comment:
wording?  "...output as soon as the server starts and has a numeric value"?

Reply:
Done.